### ISSUE TITLE
LVO-Logging ignores Loglevel.Nothing

### ISSUE THREAD

**Issue by codecoster** [LVM-IT/lvo-logging-lib#1]
# Bug Report or Feature Request (mark with an x)

```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

# Versions.
```
@angular/cli: 1.0.3
node: 6.10.3
os: darwin x64
@angular/animations: 4.1.2
@angular/common: 4.1.2
@angular/compiler: 4.1.2
@angular/core: 4.1.2
@angular/forms: 4.1.2
@angular/http: 4.1.2
@angular/platform-browser: 4.1.2
@angular/platform-browser-dynamic: 4.1.2
@angular/router: 4.1.2
@angular/cli: 1.0.3
@angular/compiler-cli: 4.1.2
```
# Repro steps.

Install @lvo/logging to your project, set the LogLevel to Nothing

```
logmanager.setCurrentLogLevel(LogLevel.Nothing);
```
The Logger will continue to generate Logging-Messages

# Desired functionality.

The Logger should stop generating logging output

**Comment by lammers** [comment]
Hi codecoster, thanks for improving things. I will provide you with a fix in the near term. 

Regards
Dirk Lammers

**Comment by lammers** [comment]
fixed in version 1.0.2
